crates/wit-bindgen-rust/tests/codegen.rs

```diff
@@ -638,7 +647,7 @@ mod example_4 {
         package example:exported-resources;
 
         world import-some-resources {
-            export logging;
+            import logging;
```

Comment:
why do we need to change this one?

Reply:
good catch, that was left over from my debugging